rootfs/api/settings/production.py

```diff
@@ -254,8 +254,8 @@
 
 # k8s image policies
 SLUGRUNNER_IMAGE = os.environ.get('SLUGRUNNER_IMAGE_NAME', 'quay.io/deisci/slugrunner:canary')  # noqa
-SLUG_BUILDER_IMAGE_PULL_POLICY = os.environ.get('SLUG_BUILDER_IMAGE_PULL_POLICY', "Always")  # noqa
-DOCKER_BUILDER_IMAGE_PULL_POLICY = os.environ.get('DOCKER_BUILDER_IMAGE_PULL_POLICY', "Always")  # noqa
+SLUG_RUNNER_IMAGE_PULL_POLICY = os.environ.get('SLUG_RUNNER_IMAGE_PULL_POLICY', "IfNotPresent")  # noqa
+IMAGE_PULL_POLICY = os.environ.get('IMAGE_PULL_POLICY', "Always")  # noqa
```

Comment:
this is the image pull policy of the user pods which uses docker file or deis pull. so i am not sure if should strict policy.